examples/redis/envoy-gateway-config.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
```

Comment:
currently we seem to be combining multiple resources into a single file
e.g. https://github.com/envoyproxy/gateway/blob/main/examples/kubernetes/grpc-routing.yaml

lets do the same here and call the file `examples/kubernetes/global-ratelimit.yaml` ?